### PR TITLE
feat(react-appkit): Add draft UI to toggle NetworkMode

### PR DESCRIPTION
**Pull request by mykola-vrmchk**
1) connecting two peers 
2) going offline with left peer
3) making concurrent offline changes
4) going back online and syncing changes

https://user-images.githubusercontent.com/36420699/209685821-4423c590-8f09-4147-9279-97e67f8996ee.mov

TODOs: https://github.com/dxos/dxos/issues/2140

### CODE REVIEW

**Comment by mykola-vrmchk** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2163** <a href="https://app.graphite.dev/github/pr/dxos/dxos/2163" target="_blank">Graphite</a> 
    * **PR #2170** <a href="https://app.graphite.dev/github/pr/dxos/dxos/2170" target="_blank">Graphite</a> 
      * **PR #2185** <a href="https://app.graphite.dev/github/pr/dxos/dxos/2185" target="_blank">Graphite</a> 
        * **PR #2186** <a href="https://app.graphite.dev/github/pr/dxos/dxos/2186" target="_blank">Graphite</a> 
          * **PR #2189** <a href="https://app.graphite.dev/github/pr/dxos/dxos/2189" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dxos/dxos/2189?utm_source=stack-comment).

File: packages/apps/patterns/react-appkit/src/components/StatusIndicator/StatusIndicator.tsx [hunk 1]
```diff
@@ -4,26 +4,45 @@
...
+          </Tooltip>
+        )}
+      </div>
+      {networkMode && (
+        <>
+          {' '}
+          <span className='ml-2'>{networkMode.mode === 0 ? 'OFFLINE' : 'ONLINE'}</span>
+          <Button
+            onClick={async () => {
+              const newMode = networkMode.mode === 0 ? 1 : 0;
```
Comment (dmaretskyi) [inline packages/apps/patterns/react-appkit/src/components/StatusIndicator/StatusIndicator.tsx:39]: whats up with enums and react?

Comment (mykola-vrmchk) [reply packages/apps/patterns/react-appkit/src/components/StatusIndicator/StatusIndicator.tsx:39]: image

Comment (mykola-vrmchk) [reply packages/apps/patterns/react-appkit/src/components/StatusIndicator/StatusIndicator.tsx:39]: That is an error from @dxos/kai

`@dxos/protocols` is included in `dependencies` in package.json
`@dxos/protocols` is also included in `optimizeDeps` in vite.config.ts

**Comment by zhenyasav** [comment]
simple airplane mode

**Pull request merged**